includes/class-admin.php

```diff
@@ -594,11 +595,21 @@ function ( $views ) {
 	 * @return array The modified actions.
 	 */
 	public static function comment_row_actions( $actions, $comment ) {
+		global $activitypub_comment_types;
+
 		if ( was_comment_received( $comment ) ) {
 			unset( $actions['edit'] );
 			unset( $actions['quickedit'] );
 		}
 
+		$comment_type = get_comment_type( $comment );
+
+		foreach ( $activitypub_comment_types as $type ) {
+			if ( $type['type'] === $comment_type ) {
+				unset( $actions['reply'] );
+			}
+		}
```

Comment:
```suggestion
	public static function comment_row_actions( $actions, $comment ) {
		if ( was_comment_received( $comment ) ) {
			unset( $actions['edit'] );
			unset( $actions['quickedit'] );
		}
		
		if ( in_array( get_comment_type( $comment ), Comment::get_comment_type_names(), true ) ) {
			unset( $actions['reply'] );
		}
```

I forgot this was a global. We have an API for it, should we use that instead?

Reply:
good point!